clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ServerGroupManagerController.java

```diff
@@ -38,13 +38,14 @@
 @RestController
 @RequestMapping("/applications/{application}/serverGroupManagers")
 public class ServerGroupManagerController {
-  final List<ServerGroupManagerProvider> serverGroupManagerProviders;
+  final List<ServerGroupManagerProvider<ServerGroupManager>> serverGroupManagerProviders;
```

Comment:
Should this instance variable (as well as the constructor parameter) be `List<ServerGroupManagerProvider<? extends ServerGroupManager>>`?

Reply:
Sure, I'm not sure it's adds much, since it's just a private field and we don't really care about any other types besides `ServerGroupManager`, but I guess it doesn't hurt...